test/RiskManagerV1.test.js

```diff
@@ -69,28 +73,181 @@ describe("RiskManagerV1", () => {
       })
     })
 
-    context("when deposit is in liquidation state", () => {
-      let notifyLiquidationTx
+    context(
+      "when deposit is in liquidation state and the surplus pool is empty",
+      () => {
+        let notifyLiquidationTx
+        let auctionAddress
 
-      beforeEach(async () => {
-        notifyLiquidationTx = await notifyLiquidation()
-      })
+        beforeEach(async () => {
+          notifyLiquidationTx = await notifyLiquidation()
 
-      it("should emit NotifiedLiquidation event", async () => {
-        await expect(notifyLiquidationTx)
-          .to.emit(riskManagerV1, "NotifiedLiquidation")
-          .withArgs(mockIDeposit.address, notifier.address)
-      })
+          auctionAddress = await riskManagerV1.depositToAuction(
+            mockIDeposit.address
+          )
+        })
 
-      it("should create an auction and populate auction's map", async () => {
-        const createdAuctionAddress = await riskManagerV1.depositToAuction(
-          mockIDeposit.address
-        )
+        it("should emit NotifiedLiquidation event", async () => {
+          await expect(notifyLiquidationTx)
+            .to.emit(riskManagerV1, "NotifiedLiquidation")
+            .withArgs(mockIDeposit.address, notifier.address)
+        })
 
-        expect(createdAuctionAddress).to.be.properAddress
-        expect(createdAuctionAddress).to.not.equal(ZERO_ADDRESS)
-      })
-    })
+        it("should create an auction ", async () => {
+          expect(auctionAddress).to.be.properAddress
+          expect(auctionAddress).to.not.equal(ZERO_ADDRESS)
+        })
+
+        it("should not use the surplus pool", async () => {
+          expect(await riskManagerV1.tbtcSurplus()).to.be.equal(0)
+        })
+      }
+    )
+
+    context(
+      "when deposit is in liquidation state and the surplus pool is " +
+        "smaller than the deposit lot size",
+      () => {
+        let notifyLiquidationTx
+        let auctionAddress
+
+        beforeEach(async () => {
+          const surplus = to1ePrecision(30, 16)
+          await testToken.mint(owner.address, surplus)
```

Comment:
We should rename `testToken` into `tbtcToken` at some point. It is confusing now.

Reply:
See https://github.com/keep-network/coverage-pools/pull/43/commits/0c6a7abca95a97ff5edd1f9fbab7302481698e8a